app/services/encrypted_document_storage/local_storage.rb

```diff
@@ -1,5 +1,10 @@
 module EncryptedDocumentStorage
   class LocalStorage
+    def read_image(name:)
```

Comment:
I'd maybe leave a comment indicating that this is intended for specs, not the normal API (like that's why the S3Storage class doesn't have a corresponding copy of this?)

```suggestion
    # Used in tests to verify results
    def read_image(name:)
```